src/frontend/typer/personopplysninger.ts

```diff
@@ -42,6 +42,49 @@ export interface IAdresse {
     type: AdresseType;
     gyldigFraOgMed?: string;
     gyldigTilOgMed?: string;
+    bostedsadresse?: IBostedsadresse;
+}
+
+export interface IBostedsadresse {
+    vegadresse?: IVegadresse;
+    matrikkeladresse?: IMatrikkeladresse;
+}
+
+export interface IVegadresse {
+    husnummer?: string;
+    husbokstav?: string;
+    bruksenhetsnummer?: string;
+    adressenavn?: string;
+    kommunenummer?: string;
+    tilleggsnavn?: string;
+    postnummer?: string;
+    koordinater?: IKoordinater;
+    matrikkelId?: IMatrikkeladresse;
+}
+
+export interface IKoordinater {
```

Comment:
Trenger vi/bruker vi alt det nye her?

Reply:
Nei, vi bruker ikke det. Gjorde det bare for å fikse warnings....